client/my-sites/plans/current-plan/jetpack-checklist/constants.js

```diff
@@ -51,11 +51,16 @@ export const JETPACK_CHECKLIST_TASKS = {
 
 export const JETPACK_CHECKLIST_TASK_AKISMET = {
 	title: translate( "We're automatically turning on spam filtering." ),
+	completedButtonText: translate( 'View spam stats' ),
 	completedTitle: translate( "We've automatically turned on spam filtering." ),
+	getUrl: siteSlug =>
+		`//${ siteSlug.replace( '::', '/' ) }/wp-admin/admin.php?page=akismet-key-config`,
```

Comment:
> I'd really discourage these one-off `slug <-> URL` transformations.

Just what I thought at first :smile: Then I searched the codebase for utils to do that kind of transformation and found none but quite a few instances of _this_ transform :sweat_smile: So I thought, fair enough, maybe that's really all there is to transforming slugs to URLs and went ahead with it :grimacing: 

> I think we're better of working from `state.sites.items[ state.ui.selectedSiteId ].options.admin_url` (maybe there's a selector for that already 🤞 )


This constants file doesn't really know about `state` and I'm not sure it's worth making it aware of it... Could blow up a bit, it's similar to something I did for the Theme Showcase. Not the part of the codebase I'm most proud of.

Reply:
> Also, this doesn't seem to have any stats about spam… It's Akismet account settings

It has a bar chart for me :thinking: I'm linking to what #32557's [illustration](https://user-images.githubusercontent.com/214813/56674490-27278e80-6688-11e9-9b36-5f377316150e.jpg) seemed to prescribe...

Reply:
I get this, maybe it's because there's no data 🤷‍♂️

<img width="1002" alt="Screen Shot 2019-05-02 at 21 39 38" src="https://user-images.githubusercontent.com/841763/57102087-ec29f880-6d22-11e9-9b7b-7577e4669923.png">

Reply:
Yeah, it'd show up above Settings:

![image](https://user-images.githubusercontent.com/96308/57127694-8163d600-6d91-11e9-9b7e-b9761f8a2e6b.png)